### PR TITLE
Add support for passing fso exec type on quicklaunch

### PR DESCRIPTION
**Pull request by Shivansps**
I finally got lazy enoght that i didnt wanted to open knossos just to work on a mod.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +28,19 @@ This greately minimizes hard drive usage, but comes with two things to keep in m
...
+If no fso-exec-type is given or "Default", or an invalid type is passed "Release" will be used instead. <br />
+Exec type options:<br />
+-Default<br />
+-Release<br />
+-Debug<br />
+-Fred2<br />
+-Fred2Debug<br />
+-QtFred<br />
+-QtFredDebug<br />
+The "Mod Settings" window displays the Command Line to play that mod directly via quick launch. Then you can use that to create a shortcut.<br />
```
Comment (wookieejedi) [inline README.md:43]: Might phrase with additional specificity: IE
"The "Settings" window on a mod displays the Command Line to play that mod directly via Quick Launch. You can copy that Quick Launch Command Line and use it to create a shortcut to easily launch that mod."

Comment (Shivansps) [reply README.md:43]: yes, can you write the entire thing?

Comment (wookieejedi) [reply README.md:43]: Yep! So Line 43 would be this: 
````
The "Settings" window on a mod displays the Command Line to play that mod directly via Quick Launch. You can copy that Quick Launch Command Line and use it to create a shortcut to easily launch that mod.
```

Comment (Shivansps) [reply README.md]: done

**Review by wookieejedi — APPROVED** [review]

**Pull request merged**